fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientSavingsIntegrationTest.java

```diff
@@ -60,7 +60,7 @@
 /**
  * Client Savings Integration Test for checking Savings Application.
  */
-@SuppressWarnings({ "rawtypes" })
+@SuppressWarnings({ "rawtypes", "AbbreviationAsWordInName" })
```

Comment:
are there many problems in this test, would it be hard to also fix this, just so that it's nice?